java/libs/src/main/java/org/physical_web/collection/PhysicalWebCollection.java

```diff
@@ -35,7 +35,7 @@
   private static final String SITEURL_KEY = "siteurl";
   private Map<String, UrlDevice> mDeviceIdToUrlDeviceMap;
   private Map<Class, UrlDeviceJsonSerializer> mUrlDeviceTypeToUrlDeviceJsonSerializer;
-  private Map<String, PwsResult> mRequestUrlToPwsResultMap;
+  private Map<String, PwsResult> mBroadcastUrlToPwsResultMap;
```

Comment:
I'm not convinced we gain much other than complexity from sharing data instead of duplicating it (and potentially swapping out old PwsResults with new ones when found).

Also, isn't it possible that the broadcast url could play into the the PwsResult in the future?  Couldn't myshadyshortener.net give a worse quality score compared to a private, trusted shortener?

Also, I'm not sure why you are saying that we generally want to look these entries up by their site url and not the broadcast url.  That isn't the case in the prototype app (nothing is ever keyed on the displayed url there).